packages/esm-patient-test-results-app/src/overview/common-datatable.component.tsx

```diff
@@ -26,6 +26,9 @@ interface CommonDataTableProps {
 }
 
 const CommonDataTable: React.FC<CommonDataTableProps> = ({ title, data, description, toolbar, tableHeaders }) => {
+  // Currently description is a complicated component requiring use of div tag, while carbon wraps this
+  // component in <p>, leading to invalid DOM nesting
+  // TODO: make a PR against carbon-components-react to allow for a tag component other than p
```

Comment:
Can you make Jira issues for your TODOs?

Reply:
Sorry to be a bureaucrat but I think it will help make sure we don't lose track of these things